sotodlib/tod_ops/demodulation.py

```diff
@@ -0,0 +1,277 @@
+"""Functions used for demodulating general sinusoidal modulation in
+timestreams. Often used in lab testing or with optical choppers."""
+
+import numpy as np
+from sotodlib.tod_ops import filters
+from scipy.optimize import curve_fit
+
+import logging
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
I realize we probably shouldn't be setting the logging level on input (did I do that in `load_smurf`? I should probably fix that...) Other two lines are good.